crates/astria-bridge-withdrawer/src/bridge_withdrawer/state.rs

```diff
@@ -39,17 +41,24 @@ macro_rules! forward_setter {
 }
 
 forward_setter!(
+    [set_startup_info <- startup::Info],
+    [set_cometbft_connected <- bool],
     [set_sequencer_connected <- bool],
     [set_last_rollup_height_submitted <- u64],
     [set_last_sequencer_height <- u64],
     [set_last_sequencer_tx_hash <- tendermint::Hash],
 );
 
 #[derive(Clone, Debug, Default, PartialEq, Eq, serde::Serialize)]
+// This lint is allowed because there are 4 boolean fields that are necessary to track here.
+#[allow(clippy::struct_excessive_bools)]
 pub(crate) struct StateSnapshot {
+    startup_info: Option<startup::Info>,
+
     watcher_ready: bool,
     submitter_ready: bool,
 
+    cometbft_connected: bool,
```

Comment:
what is this for? right now, the only connection to the sequencer is via cometbft?

Reply:
this is my bad, had some git troubles and this slipped past me

Reply:
Should probably disambiguate between `sequencer_cometbft_connected` and `sequencer_grpc_connected` or something like that. As it stands, this is a bit confusing.

Reply:
@itamarreif can you remove this then?